lib/install/management/inspect.go

```diff
@@ -89,11 +89,11 @@ func (d *Dispatcher) InspectVCH(vch *vm.VirtualMachine, conf *config.VirtualCont
 		log.Debugf("No host certificates provided")
 	}
 
-	d.ShowVCH(conf, "", "", "", "")
+	d.ShowVCH(conf, "", "", "", "", "")
```

Comment:
Is there any way we can guess at a cert path and then in ShowVCH check the certs against the VCH for validity? As it stands `inspect` is never going to provide useful output for secure deployments.

We could check:
default cert-path location
DOCKER_CERT_PATH location

for existing certs.